Testscripts/Windows/VMRESIZE-HB.ps1

```diff
@@ -0,0 +1,255 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+<#
+.Synopsis
+	Perform a simple VM hibernation in Azure before VM resizng
+	VM resizing reboots the VM and fails the hibernation resume.
```

Comment:
Could you please clarify more for what is expected behavior?

Reply:
Updated.